Import-GDx.ps1

```diff
@@ -94,7 +104,7 @@ function ConvertTo-Jpg {
     param (
         # File name to convert
         [Parameter(Mandatory=$true)]
-        [ValidateScript({Test-Path -PathType Leaf -Include "*.png" $_})]
+        [ValidateScript({$DryRun -or (Test-Path -PathType Leaf -Include "*.png" $_)})]
```

Comment:
Fixes an issue where dry-runs wouldn't work, because the parameter validation would fail here